spec/factories/company_users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :company_user do
+    company { nil }
```

Comment:
FactoryBot knows when the model has a belongs_to association. You can just let this implicit
```suggestion
    company
```

Reply:
?